src/js/util/index.js

```diff
@@ -384,3 +387,24 @@ export const makeSelection = selectedDiv => {
   range.selectNode(selectedDiv);
   window.getSelection().addRange(range);
 };
+
+/**
+ * Manipulates the date string
+ * @param {string} inputDate
+ */
+export const dateMath = (inputDate, dateFormat) => {
+  const d = moment(inputDate, dateFormat);
+  return {
+    add: (length, unit) => d.add(length, unit).format(dateFormat),
+    subtract: (length, unit) => d.subtract(length, unit).format(dateFormat),
+    changeFormat: newFormat => d.format(newFormat),
+    isBefore: date => {
+      const newDate = moment(date, dateFormat);
+      return d.isBefore(newDate);
+    },
+    isAfter: date => {
+      const newDate = moment(date, dateFormat);
+      return d.isAfter(newDate);
+    },
+  };
+};
```

Comment:
https://github.com/you-dont-need/You-Dont-Need-Momentjs/blob/master/README.md#add